### ISSUE TITLE
Add some text about converters when using @Consumes

### ISSUE THREAD

**Issue by linyatis** [caelum/vraptor4#874]
There is no text about how to convert an object when using `@Consumes`.
Docs just informs about converting plain-text requests.

**Comment by Turini** [comment]
@linyatis let's do it together? :)

**Comment by linyatis** [comment]
Sure, I'm on :)

**Comment by linyatis** [comment]
@Turini, where should we put it?  
I'm thinking about 2 pages there: [Converters][1] or the REST Controllers (section [Consuming in other formats][2]).

I believe that the best place is at the Converters' page. We could explain that we have two types of converters: one used when you do a form request and another when you do a xmlHttpRequest (sorry if I use any incorrect term here).

[1]:http://www.vraptor.org/en/docs/converters/
[2]:http://www.vraptor.org/en/docs/controllers-rest/#consuming-in-other-formats

**Comment by Turini** [comment]
we can add on the converters' page, just like you said, and add a paragraph 
at the end of "consuming in other formats" section, linking to it. What u think?

**Comment by linyatis** [comment]
I agree with you. I can write something here and do a PR. What do you think?

**Comment by Turini** [comment]
Amazing! Let me know if you need any help to do that :)
